ui/shared/StyledTArea.qml

```diff
@@ -0,0 +1,8 @@
+import QtQuick 2.13
```

Comment:
We already have a StyledTextArea: https://github.com/status-im/nim-status-client/blob/master/ui/shared/StyledTextArea.qml
Unless you want it to be the default one without the background?

Reply:
Yeah, I need the default one similar to how it was done here: https://github.com/status-im/nim-status-client/blob/master/ui/shared/StyledTextEdit.qml